Makefile

```
@@ -53,14 +64,17 @@ test-lint: ## Enforce test architecture (ADR-130) — runs before tests
 test-fast: $(BUILD)/test_toml ## Run fastest tests only (<2s)
 	./$(BUILD)/test_toml
 
-test-unit: $(BUILD)/test_toml $(BUILD)/test_checks $(BUILD)/test_version $(BUILD)/test_rules $(BUILD)/test_sort $(BUILD)/test_tokenizer $(BUILD)/test_import_graph ## Run unit tests
+test-unit: $(BUILD)/test_toml $(BUILD)/test_checks $(BUILD)/test_version $(BUILD)/test_rules $(BUILD)/test_sort $(BUILD)/test_tokenizer $(BUILD)/test_import_graph $(BUILD)/test_dup_symbols $(BUILD)/test_platform $(BUILD)/test_runner ## Run unit tests
```

checkmake (0.3.2)

[warning] Target body for "test-unit" exceeds allowed length of 5 lines (10).

(maxbodylength)